dataland-backend/src/main/kotlin/org/dataland/datalandbackend/model/StorageHashMap.kt

```diff
@@ -0,0 +1,11 @@
+package org.dataland.datalandbackend.model
+
+import org.springframework.stereotype.Component
+
+/**
+ * Implementation of a temporary storage business data to be inserted into the internal storage via Dataland
+*/
+@Component
+class StorageHashMap {
```

Comment:
_if_ we need this, what about:
instead of 
```
@Component
class StorageHashMap {
    var map = HashMap<String, String>()
}
```

```
@Component
class StorageHashMap: HashMap<String, String>()
```

Reply:
WIP

Reply:
Done. It's gone and just a member of the DataManager

Reply:
ok